ansible_collections/arista/avd/plugins/modules/eos_validate_state_runner.py

```diff
@@ -12,64 +12,80 @@
 module: eos_validate_state_runner
 version_added: "4.4.0"
 author: Arista Ansible Team (@aristanetworks)
-short_description: Leverage ANTA for eos_validate_state role
+short_description: Utilizes ANTA for eos_validate_state role validation
```

Comment:
eos_validate_state role validation is a mouthful .. not sure how to reword this

```suggestion
short_description: Use ANTA for eos_validate_state role validation
```

Reply:
Fixed in 68479d7a6549495d8a3a4e5ce970b358e51ea5bc